Rakefile

```diff
@@ -15,11 +15,10 @@ RDoc::Task.new(:rdoc) do |rdoc|
 end
 
 APP_RAKEFILE = File.expand_path("../test/dummy/Rakefile", __FILE__)
-load 'rails/tasks/engine.rake'
-
-
-load 'rails/tasks/statistics.rake'
 
+# FIXME: these directories/files are not present in the gem directory...
+# load 'rails/tasks/engine.rake'
+# load 'rails/tasks/statistics.rake'
```

Comment:
I wasn't sure what to do with these, they caused a "file not found" error.

Reply:
Removing these lines (loading nonexistent files) did not seem to have any adverse effect. 
I believe PR is ready for review.